functional-tests/bdd-tests/CRS_pay_licence_fee.feature

```diff
@@ -0,0 +1,14 @@
+﻿Feature: CRS_pay_licence_fee
+    As a logged in business user
+    I want to pay the Cannabis Retail Store fee
+
+Scenario: Pay CRS Licence Fee
+    Given the CRS application has been approved
+    And I am logged in to the dashboard as a private corporation
+    And I click on the Licences tab
+    And I click on the Pay Licence Fee and Plan Store Opening link
+    And I enter the estimated opening date and the opening date reason
+    And I click on the payment button
+    And I complete the payment
+    And I click on the Licences tab
+    Then the Licences tab has been updated
```

Comment:
does "Updated" have a description? for example:
* the expiry date
* licence download link
* change jobs available.. etc? 
Or is it enough to say updated?